### ISSUE TITLE
tests failing

### ISSUE THREAD

**Issue by JoshZeidner** [beakerbrowser/beaker#724]
Operation System: OSX

I get the following error messages when I run **npm test**.

```



> beaker-browser-test@0.0.0 test /Users/jjzeidner/electron/beaker/tests
> ava ./*-test.js -s

/Users/jjzeidner/electron/beaker/tests/node_modules/normalize-package-data/lib/fixer.js:191
      throw new Error('Invalid version: "'+ data.version + '"')
      ^

Error: Invalid version: "https://registry.npmjs.org/ava/-/ava-0.16.0.tgz"
    at Object.fixVersionField (/Users/jjzeidner/electron/beaker/tests/node_modules/normalize-package-data/lib/fixer.js:191:13)
    at /Users/jjzeidner/electron/beaker/tests/node_modules/normalize-package-data/lib/normalize.js:32:38
    at Array.forEach (<anonymous>)
    at normalize (/Users/jjzeidner/electron/beaker/tests/node_modules/normalize-package-data/lib/normalize.js:31:15)
    at module.exports (/Users/jjzeidner/electron/beaker/tests/node_modules/meow/index.js:47:2)
    at Object.<anonymous> (/Users/jjzeidner/electron/beaker/tests/node_modules/ava/cli.js:52:11)
    at Module._compile (module.js:624:30)
    at Object.Module._extensions..js (module.js:635:10)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
    at Function.Module.runMain (module.js:665:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
npm ERR! Test failed.  See above for more details.



```

**Comment by pfrazee** [comment]
I'm not sure what's causing this. Maybe try reinstalling the node modules in the tests dir? Or maybe check which version of ava you have installed globally

**Comment by him2him2** [comment]
This seems to be a problem others have had with normalize-package-data. I found a [solution](https://forums.meteor.com/t/solved-npm-invalid-version-error-preventing-galaxy-deployment/38150) that worked for me.



I resolved this issue by deleting my node_modules directory in "tests" directory and then running install.

you will need to **cd beaker/tests** directory, then: 
```
rm -R node_modules
npm install
```
